metadata/2024-07-09-enable-aggregator-v2-is-at-least-api/enable_aggregator_v2_is_at_least_api.json

```diff
@@ -0,0 +1,6 @@
+{
+  "title": "Introduce native API to evaluate aggregator value is greater or equal to given amount",
+  "description": "Enables the changes in AIP-87 https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-87.md",
+  "source_code_url": "https://github.com/aptos-labs/aptos-core",
```

Comment:
source code url need to be the code to the .move script, so if people want to compute the proposal hash, they can do that